Tiltfile

```diff
@@ -0,0 +1,5 @@
+
+# If not using a standard local dev name, specify your k8s context here
+#allow_k8s_contexts('jenkins-dev')
+k8s_yaml(helm('charts/jenkins', values='charts/jenkins/values.yaml', set=['master.adminPassword=jenkins']))
```

Comment:
I think the `adminPassword` is the only override value that I was setting in the dev yaml file that is really critical. Otherwise, you have to do some `kubectl` wizardry to fetch the randomly generated password every time you regenerate the helm deployment.